src/App.tsx

```diff
@@ -15,71 +15,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const goodsList = goods.map(good => (
```

Comment:
Really interesteting you save the whole markup to a variable.

Reply:
yeah, the magic of react. Still, you shouldn't do it, it decreases readability